Formula/apache-flex-sdk.rb

```diff
@@ -0,0 +1,49 @@
+class ApacheFlexSdk < Formula
+  desc "Framework for building expressive web and mobile applications"
+  homepage "https://flex.apache.org/"
+  url "http://mirror.fibergrid.in/apache/flex/installer/3.2/apache-flex-sdk-installer-3.2.0-src.tar.gz"
+  version "3.2.1"
+  sha256 "f8fb00d300c74609771b23e2bc5d9a97e22f793518245be5596d9a4177236f9e"
+
+  option "with-playerglobal", "Installs PlayerGlobal.swc"
```

Comment:
I don't know what it does, but it says that it's required by the SDK as other things are required as well.
```
Note: the installer downloads other files besides the Apache Flex SDK. These files are needed for the proper functioning of the SDK. The required TLF and the OSMF frameworks, as well as the optional Embedded Font Libraries and Utilities and BlazeDS are mentioned separately in the installer application. The other non-Apache files downloaded by the installer are:

Adobe AIR SDK
PlayerGlobal.swc
SWFObject.js
```